modules/i18n/src/translations/en-US/portals/myaccount.ts

```diff
@@ -1720,8 +1720,8 @@ export const myAccount: MyAccountNS = {
     },
     sections: {
         accountRecovery: {
-            description:
-            "Manage recovery information that we can use to help you recover your password",
+            description: "Manage recovery information that we can use to help you recover your password",
```

Comment:
Shall we check and update the other language localizations of this `description` if needed?

Reply:
The text is not updated, just fixed a spacing issue.